modules/core-dev/manifests/init.pp

```diff
@@ -39,6 +39,25 @@
 		require => Exec['git_exclude_exists']
 	}
 
+	exec { 'maybe_checkout_wp_develop':
+		command => '/usr/bin/git clone git://develop.git.wordpress.org/ /vagrant/wordpress-develop',
+		creates => '/vagrant/wordpress-develop'
```

Comment:
I think using the vcsrepo puppet module gives us more control, here; willing to be argued on that point, though, as it does introduce a new submodule. See #15.